python/lsst/jointcal/jointcal.py

```diff
@@ -885,15 +884,24 @@ def _load_data(self, inputSourceTableVisit, inputVisitSummary, associations,
             catalogMap = {ref.dataId['visit']: i for i, ref in enumerate(inputSourceTableVisit)}
             detectorDict = {detector.getId(): detector for detector in camera}
 
+            columns = None
```

Comment:
Please add a note here about how this is filled out with the columns to be load from the sourceTable during the first loop iteration.